tests/providers/google/cloud/operators/test_dataflow.py

```diff
@@ -540,8 +559,9 @@ def test_validation_deferrable_params_raises_error(self):
             DataflowTemplatedJobStartOperator(**init_kwargs)
 
     @pytest.mark.db_test
-    @mock.patch("airflow.providers.google.cloud.operators.dataflow.DataflowHook.start_template_dataflow")
-    def test_start_with_custom_region(self, dataflow_mock):
+    @mock.patch(f"{DATAFLOW_PATH}.DataflowTemplatedJobStartOperator.xcom_push")
+    @mock.patch(f"{DATAFLOW_PATH}.DataflowHook.start_template_dataflow")
+    def test_start_with_custom_region(self, dataflow_mock, mock_xcom_push):
```

Comment:
We're mocking xcom_push here. Should we assert it's called or not?

Reply:
I did not think that we should assert if every mock was called or not, and added it just to make the test pass. Since we're not testing xcom_push anywhere else, I think I could add the call assert. Thanks!